### ISSUE TITLE
what Demo

### ISSUE THREAD

**Issue by Gamer1990** [tgudelj/MATRICIntegrationDemo#4]
im missing something i have api enable by gui but the "Demo" file is Corrupt or some thing its not runing and i cant find "demo"

**Comment by tgudelj** [comment]
Did you import IntegrationDemo.deck to your MATRIC server (just double click)?

**Comment by Gamer1990** [comment]
yes i did import it but a month after this i was in a life Alternating Accident ie lost my job and as such i dont program as much as i was
